packages/react-jss/src/withStyles.js

```diff
@@ -39,6 +39,29 @@ const getStyles = <Theme: {}>(styles: Styles<Theme>, theme: Theme) => {
   if (typeof styles !== 'function') {
     return styles
   }
+  if (process.env.NODE_ENV !== 'production') {
+    if (styles.length < 1) {
+      console.warn(
```

Comment:
warning(false, 'error')

Reply:
why `false`?

Reply:
I thought it would be something `warning(process.env.NODE_ENV !== 'production' && styles.length < 1, 'ERROR')

Reply:
no, it is designed to use like this `warning(isValueCorrect, 'error')` so if value is correct no warning happens

Reply:
like an assertion

Reply:
warning module does the check for env, but we want to avoid the .length check in production here, so we should have the production cheeck like you did

Reply:
I think the warning calls are wrapped automatically with the env check now.

Reply:
Also `!== 1` please

Reply:
yes, but we don't need the .length check in runtime either, I am not sure its cheap

Reply:
actually we can check, lets see quickly a bench

Reply:
https://esbench.com/bench/5c5078f94cd7e6009ef6209d

ok I think its fine to use fn.length at runtime, so we can just write warning(styles.length === 1, 'error') without adding env checks

Reply:
this warning will be stripped away from prod build anyway, right?

Reply:
Yeah, all of the warning calls are automatically wrapped with `process.env.NODE_ENV === 'development'`.

Reply:
Just checked, yes it gets removed in the minified version, even if it isn't wrapped with evn conditional. Not sure who removes it though :)

Reply:
All of the warning come this in the bundled code:
```
if (process.env.NODE_ENV !== 'production') {
  warning();
}
```

In the minified umd version this will be removed because we replace the node env. In the commonjs and ESM, it will exist until the user bundles the code.

Reply:
true,  something like this

```
        warning(false, "[JSS] Bad keyframes name " + key);
```

becomes in the ESM bundle

```
      process.env.NODE_ENV !== "production" ? warning(false, "[JSS] Bad keyframes name " + key) : void 0;

```

so later uglify probably removes it once conditional becomes 'production' !== 'production'